package.json

```diff
@@ -74,16 +77,15 @@
     "@types/react-dom": "18.3.0",
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@types/react-window": "^1.8.5",
-    "@typescript-eslint/eslint-plugin": "7.18.0",
-    "@typescript-eslint/parser": "7.18.0",
     "beachball": "^2.33.2",
-    "eslint": "8.57.0",
-    "eslint-config-prettier": "9.1.0",
-    "eslint-plugin-import": "2.31.0",
-    "eslint-plugin-playwright": "^0.15.3",
+    "eslint": "9.18.0",
+    "eslint-config-prettier": "10.0.1",
+    "eslint-plugin-import": "^2.31.0",
```

Comment:
the rule is to not use carets in root package.json -> please update to fixed versions